src/neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -18,6 +18,7 @@ partial class ApplicationEngine
         public static readonly InteropDescriptor System_Contract_Create = Register("System.Contract.Create", nameof(CreateContract), 0, CallFlags.AllowModifyStates, false);
         public static readonly InteropDescriptor System_Contract_Update = Register("System.Contract.Update", nameof(UpdateContract), 0, CallFlags.AllowModifyStates, false);
         public static readonly InteropDescriptor System_Contract_Destroy = Register("System.Contract.Destroy", nameof(DestroyContract), 0_01000000, CallFlags.AllowModifyStates, false);
+        public static readonly InteropDescriptor System_Contract_Exists = Register("System.Contract.Exists", nameof(ContractExists), 0_01000000, CallFlags.AllowCall, false);
```

Comment:
`Method.Exists`?

Reply:
My goal it's to be able to check if a contract or method exists, if we use `null` for `method`, should we use two methods? `Contract.Exists` and `Method.Exists`?

Reply:
If you want to check if the contract exists, you can use `System.Blockchain.GetContract`.

Reply:
> System.Blockchain.GetContract

Strange, rename it `System.Contract.GetContract`?

Reply:
> My goal it's to be able to check if a contract or method exists, if we use `null` for `method`, should we use two methods? `Contract.Exists` and `Method.Exists`?

And we can extend it in devpack, split into two methods: `Contract.Exits(hash)` and `Contract.Exists(hash, method)`.

Reply:
Done